variables.tf

```diff
@@ -55,6 +43,58 @@ variable "access_tags" {
   }
 }
 
+##############################################################################
+# Naming Variables
+##############################################################################
+
+variable "prefix" {
+  description = "The value that you would like to prefix to the name of the resources provisioned by this module. Explicitly set to null if you do not wish to use a prefix."
+  type        = string
+  default     = null
+}
+
+variable "name" {
+  description = "The string to use for the naming of VPC, when var.create_vpc is true. This string is also used as a prefix for the naming of VPC resources."
```

Comment:
If prefix value is not passed, then the value of this variable is exactly what the VPC will be called.